setup.cfg

```diff
@@ -22,7 +22,7 @@ classifers =
 [options]
 packages = find:
 python_requires = >= 3.8
-
+    
```

Comment:
Super minor - can you undo this whitespace change?

Reply:
Sorry, missed your comment